pkg/context/httpcontext.go

```diff
@@ -59,6 +60,12 @@ type (
 
 		OnFinish(FinishFunc) // For setting final client statistics, etc.
 		AddTag(tag string)   // For debug, log, etc.
+		// Update next lazy tag, without creating new object if possible.
+		AddLazyTag(ns string, prefix string, msg string, intMsg int)
+		// Converts LazyTags to string and return them with normal tags.
+		GetTags() []string
+		// Recycle LazyTags back to lazyTagPool.
+		ReleaseLazyTags()
```

Comment:
Propose to change `GetTags`, `ReleaseLazyTags` to be `getTags`, `releaseLazyTags`. these methods are used for HTTPContext to debug. They may not allowed to be called outside. I think filters should not call these methods...

Reply:
Good point, thanks!